telemetry/specs/INT.mdk

```diff
@@ -384,12 +384,36 @@ ignore the designated bit position.)
 * TCP/UDP destination Port field: A port number in layer 4 will be reserved to
 indicate the existence of INT after TCP/UDP. INT source changes the destination
 Port, and sink must restore the original port number.
+* New Probe Marker fields: arbitrary 64-bit values are inserted after IP TCP/UDP header
+to indicate the existence of INT after TCP/UDP. These fileds should be interpreted
+as unsigned integer values, stored in network byte order and are initialized to a 
+configured value.  This Probe marker is a variation of an early IETF draft
+https://tools.ietf.org/html/draft-lapukhov-dataplane-probe-01 (expired), with existing
+implemetations.
 
-The decision to use DSCP or destination port field and the reserved values are
+INT probe marker for TCP/UDP:
+
+    0                   1                   2                   3
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                         Probe Marker (1)                      |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                         Probe Marker (2)                      |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+
+The decision to use DSCP, destination port field or probe marker, and 
+the reserved or configured values are
 per domain decisions that all INT capable devices should follow. (Note: in
 general, encoding into DSCP field will be less intrusive compared to changing
 the layer 4 port field. The latter may alter ECMP behavior and can complicate
-ACL and network debugging.)
+ACL and network debugging. If DSCP field cannot be reseved for INT, and 
+retaining ECMP behavior is desired, adding probe marker fields could be an
+option. With arbitary values being inserted after TCP/UDP header, the chance
+to conflict with user traffic in a data center is low, but cannot be completely 
+eliminated. A user could choose to combine port number along with 64-bit marker
+as an INT indicator to reduce the conflicting even more.) 
+
```

Comment:
Perhaps leave out the last sentence. If a destination port number is dedicated to INT, what is the use-case to add 64-bit marker also? The only reason to do so would be if some switches in the network recognized INT destination port, but some other switches only recognized probe marker. But it is understood that if there is a mix of switches that use different mechanisms for identifying INT, then any of these three approaches can be combined together.

Reply:
This extra "port" check applies to only "probe marker" option, shouldn't impact non-probe-marker approach. I will re-word it.

Reply:
Thanks for clarifying. I  misunderstood this to mean a combination of destination port number approach to identifying INT and probe marker approach to identify INT. Please reword. JK/I will add clarification regarding combinations of approaches.